### PR TITLE
PORTALS-1497: add demo for CardContainerLogic component, and demo empty results

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Screen Shot 2021-02-02 at 2 48 10 PM

File: src/lib/containers/CardContainer.tsx [hunk 1]
```diff
@@ -98,7 +98,14 @@ export const CardContainer = (props: CardContainerProps) => {
     // data was retrieved from the backend but there is none to show, we return a empty fragment
     // there could be a more informed UX decision here but the current use case right now
     // should display nothing
```
Comment (nickgros) [inline src/lib/containers/CardContainer.tsx:100]: I think you can get rid of this above comment 🙂

File: src/lib/style/base/_core.scss [hunk 2]
```diff
@@ -910,6 +910,11 @@ div.SRC-userImg {
   font-size: 21px;
   margin: 20px 0px;
 }
+.SRC-no-results-title {
+  font-weight: bold;
+}
+// .SRC-no-results-description {  
+// }
```
Comment (nickgros) [inline src/lib/style/base/_core.scss:917]: Remove unused SCSS

**Review by nickgros — APPROVED** [review]
Looks good!

File: src/lib/containers/CardContainer.tsx (hunk 1 above)
Comment (jay-hodgson) [reply src/lib/containers/CardContainer.tsx]: Nice catch, updated comment.

File: src/lib/style/base/_core.scss (hunk 2 above)
Comment (jay-hodgson) [reply src/lib/style/base/_core.scss]: Removed.

**Pull request merged**